PurchasesTests/Identity/IdentityManagerTests.swift

```diff
@@ -42,6 +42,10 @@ class IdentityManagerTests: XCTestCase {
                                                                systemInfo: systemInfo)
     }
 
+}
+
+extension IdentityManagerTests {
```

Comment:
I like moving helper methods into extensions, but I think this is arbitrary to comply with `type_body_length`, so I'd exclude that in tests too.